rvic/parameters.py

```diff
@@ -122,6 +122,9 @@ def parameters(config, numofproc=1):
         gen_uh_final(outlets, dom_data, config_dict, directories)
         # ---------------------------------------------------------------- #
 
+    except BaseException as e:
```

Comment:
Again, I think that this is essentially a no-op. Can you explain the value-add here?